<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HLM PV Import Service Manager</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <item>
         <property name="text">
          <string>test</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>test2</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_2">
        <row>
         <property name="text">
          <string>Row1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Row2</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Col1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Col2</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>c</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>d</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>a</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>b</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="serviceLogLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Service Log Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="serviceLogText">
        <property name="font">
         <font>
          <family>Lucida Console</family>
         </font>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionGeneral"/>
    <addaction name="separator"/>
    <addaction name="actionChannel_Access"/>
    <addaction name="separator"/>
    <addaction name="actionDB_Connection"/>
    <addaction name="separator"/>
    <addaction name="actionService_Directory"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionDebug_log"/>
    <addaction name="separator"/>
    <addaction name="actionService_directory"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuOpen"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionDB_Connection">
   <property name="text">
    <string>DB Connection</string>
   </property>
  </action>
  <action name="actionasdd">
   <property name="text">
    <string>asdd</string>
   </property>
  </action>
  <action name="actionabc">
   <property name="text">
    <string>abc</string>
   </property>
  </action>
  <action name="actionasd">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
  <action name="actionasd_2">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
  <action name="actionasd_3">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
  <action name="actionOpen_debug_log">
   <property name="text">
    <string>Open debug log</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionGeneral">
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actionChannel_Access">
   <property name="text">
    <string>Channel Access</string>
   </property>
  </action>
  <action name="actionService_Directory">
   <property name="text">
    <string>Service Directory</string>
   </property>
  </action>
  <action name="actionDebug_log">
   <property name="text">
    <string>Debug log</string>
   </property>
  </action>
  <action name="actionService_directory">
   <property name="text">
    <string>Service directory</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
